scripts/build.py

```diff
@@ -60,6 +82,20 @@ def main():
             action='store_true',
             dest='win32')
 
+    for lang in optional_languages:
```

Comment:
You could use argparse to enforce mutual exclusion:

https://docs.python.org/3/library/argparse.html#mutual-exclusion

Reply:
Ah, yes of course.

I think I initially considered something like `--enable-lang=go`, which would build up a list and then need checking later, but with `--require-go` and `--disable-go` the mutually exclusive groups work fine.

I'll make the change.

Reply:
I'm happy to merge without, just feels a bit more idiomatic.

Reply:
I agree. I'll change it.